build-support/native-image/find-pants-targets-from-macro-decls.coffee

```diff
@@ -0,0 +1,14 @@
+byline = require 'byline'
+
```

Comment:
Can we do this in python, rust, or bash instead? I don't think that introducing a new language here will help with handoff of this feature.